src/QuantumObjects.jl

```diff
@@ -1,18 +1,26 @@
 module QuantumObjects
 import Base: *, ∘, kron, adjoint
 import LinearAlgebra: Hermitian, tr
-export QuantumObject, Ket, Bra, DensityMatrix, inner
+export QuantumObject, Ket, Bra, DensityMatrix, inner,norm
 
 abstract type QuantumObject end
 
+norm(coefficients::Array{ComplexF64,1})  = sqrt(sum((abs.(coefficients)).^2))
```

Comment:
Do not redefine function from LinAlg

Reply:
This is not resolved. 
Import norm from LinearAlgebra, add a method definition for the different objects.